ui/src/Citation.tsx

```diff
@@ -13,31 +13,41 @@ export class Citation extends React.PureComponent<CitationProperties> {
       <ScholarReaderContext.Consumer>
         {({ papers, setJumpPaperId, setSelectedCitation }) => {
           const paper = papers[this.props.citation.paper];
-          return (
-            <div className="citation">
-              <p className="title">{paper.title}</p>
-              {paper.authors.length > 0 && (
-                <>
-                  by {paper.authors[0].name}{" "}
-                  {paper.authors.length > 1 && " et al."}
-                </>
-              )}
-              {paper.abstract !== null ? (
-                <div className="citation__abstract">
-                  {truncateText(paper.abstract, 400)}
-                  <span
-                    className="citation__abstract__sidebar-link"
-                    onClick={() => {
-                      setSelectedCitation(this.props.citation);
-                      setJumpPaperId(this.props.citation.paper);
-                    }}
-                  >
-                    (see more)
-                  </span>
-                </div>
-              ) : null}
-            </div>
-          );
+          if (paper === undefined) {
```

Comment:
@andrewhead wondering your thoughts on either sticking to var == null or var === undefined || var === null. I think we should maintain consistency across the UI as far as null checks :o.

Reply:
Good thinking, I'm always all for consistency! I think my (albeit weak) preference is to have explicit, strict-equality checks (`===`) for any types that the type definitions say are possible. In this case, the only value we would have to check against according to type definitions would be `undefined` (i.e. what you get when you look in a JavaScript object for a nonexistent property). What do you think of that policy? Is there a good argument in favor of the `var == null` check?

Reply:
No, i don't think so hahah (not unless anyone explicitly passes in null). Just wanted to be on the same page, since I've been leaning the var == null route haha